dynamodb-enhanced/runtime/src/main/java/io/quarkus/amazon/dynamodb/enhanced/runtime/DynamodbEnhancedClientRecorder.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkus.amazon.dynamodb.enhanced.runtime;
+
+import io.quarkus.arc.runtime.BeanContainerListener;
+import io.quarkus.runtime.annotations.Recorder;
+import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+@Recorder
+public class DynamodbEnhancedClientRecorder {
+
+    public BeanContainerListener setDynamoDbClient() {
+        return beanContainer -> {
```

Comment:
Let's avoid lambdas in runtime code. They have a cost. It's fine for processors but we try to limit them in runtime code to places where they really are needed.

Reply:
done